com.lablicate.OpenChrom.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.0-->
+  <id>com.lablicate.OpenChrom</id>
+  <name>OpenChrom</name>
+  <summary>Open Source Chromatography Data System</summary>
+  <developer_name>Lablicate GmbH</developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>EPL-1.0 AND LicenseRef-proprietary=https://lablicate.com/about/terms</project_license>
+  <description>
+    <p>OpenChrom® is a vendor independent open source software for chromatography, spectrometry and spectroscopy. It has a strong focus on chromatography and nominal mass spectrometry, but next to GC/MS and GC/FID it has basic support for HPLC-UV/VIS, FTIR, PCR and NMR.</p>
+    <p>The base version is free and open source software with most of it curated under the Eclipse ChemClipse umbrella. Its numerous carefully reverse-engineered file format converters for vendors like Agilent, Thermo, PerkinElmer, Shimadzu, Varian, Bruker, ChromTech, Thermo/Finnigan, Waters, CAMAG, ABSciex, MassFinder, EZChrom... are however closed source and have to be installed at runtime accepting the proprietary license terms.</p>
+    <p>Moreover the Lablicate marketplace allows for the installation of proprietary databases and plug-ins to speed up workflows. These are reserved for commercial customers. Students with a valid university e-Mail can apply for significant discounts on the Lablicate portfolio.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://lablicate.com/static/media/openchrom.88bf4b9f.png</image>
```

Comment:
it's one `image` tag per `screenshot`. That's part of the build failure

it is also missing the `launchable`